### PR TITLE
js: Remove Container.replicate()

### PR DESCRIPTION
**Pull request by kayousterhout**
This has been removed from all of the blueprints we maintain,
so is now ready to be removed from Quilt. We're removing this
because the semantics of it are confusing, and it's easy for
users to loop to create many containers.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Release 0.4.0 makes some minor UX improvements.
 
 - Check for unexpected keys in the optional argument passed to the Machine,
 Container, and Deployment constructors.
+- Remove the Container.replicate() method.
```
Comment (kklin) [inline CHANGELOG.md]: This should go under `Up Next`, and probably under a `Breaking Changes` section.

**Comment by kklin** [comment]
Also, food for thought for the CI/CD deployment for our docs -- this change should also trigger the docs to redeploy.

**Comment by luise** [comment]
Oh, and when you're at it, can you also update the website example to `quilt run ./meanExample.js` instead of `quilt run ./mean.js` (https://github.com/quilt/mean/pull/10)?

File: CHANGELOG.md (hunk 1 above)
Comment (kayousterhout) [reply CHANGELOG.md]: Ah sorry about this sloppiness

**Comment by kayousterhout** [comment]
I updated this and also the website: https://github.com/quilt-corp/quilt-website/pull/3

File: README.md [hunk 2]
```diff
@@ -45,8 +45,11 @@ Load balance traffic.
 
 [//]: # (b1)
 ```javascript
-let webGroup = web.replicate(3);
-let webLoadBalancer = new LoadBalancer('web-lb', webGroup); // A load balancer over 3 containers.
+let webContainers = [];
+for (i = 0; i < 3; i +=1) {
```
Comment (kklin) [inline README.md]: There should be a space after the `+=`

**Comment by kayousterhout** [comment]
Fixed

**Pull request merged**